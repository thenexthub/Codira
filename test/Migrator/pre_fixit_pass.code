// REQUIRES: objc_interop
// RUN: %empty-directory(%t) && not %target-language-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/Inputs/API.json -emit-migrated-file-path %t/pre_fixit_pass.code.result -o /dev/null
// RUN: diff -u %S/pre_fixit_pass.code.expected %t/pre_fixit_pass.code.result

import Bar

struct New {}
@available(*, unavailable, renamed: "New")
struct Old {}
Old()

fn foo(_ a : PropertyUserInterface) {
  a.setField(1)
  a.setURL(1)
  _ = a.field()
}
