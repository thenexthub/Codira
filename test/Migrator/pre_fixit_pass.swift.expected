// REQUIRES: objc_interop
// RUN: %empty-directory(%t) && not %target-language-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/Inputs/API.json -emit-migrated-file-path %t/pre_fixit_pass.code.result -o /dev/null
// RUN: diff -u %S/pre_fixit_pass.code.expected %t/pre_fixit_pass.code.result

import Bar

struct New {}
@available(*, unavailable, renamed: "New")
struct Old {}
New()

fn foo(_ a : PropertyUserInterface) {
  a.field = 1
  a.url = 1
  _ = a.field
}
